accepted/0022-git-buildpack.md

```diff
@@ -0,0 +1,61 @@
+# Git Support
+
+## Proposal
+
+Introduce an (optional) Git buildpack into each language buildpack family. The buildpack
+will be charged with providing `git` metadata as build-time and run-time environment
+variables.
+
```

Comment:
```suggestion

This buildpack would be maintained by the existing Tooling team.
```

Reply:
@genevieve can you include this line so that we can record ownership for the new repo.